solidus_volume_pricing.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  s.add_runtime_dependency 'solidus_core', '~> 1.3'
+  s.add_runtime_dependency 'solidus_core', '~> 2.1'
```

Comment:
If the only change necessary was using `routes.draw`, can this extension not depend on solidus `>= 1.3` instead? That way, you can also keep the build matrix for 1.3 and 1.4.